data/bedrock/latest/types.yml

```diff
@@ -1169,7 +1175,31 @@ TrackedObject:
 # MapDecoration is a fixed decoration on a map: Its position or other properties do not change automatically
 # client-side.
 MapDecoration:
-   type: u8
+   type: u8 =>
```

Comment:
It's ok if you want to do these changes, but if done in this PR then we'd need to add to all the older versions as well